README.md

```diff
@@ -10,7 +10,7 @@
   *  This library is being maintained by the team at [NameHash Labs](https://namehashlabs.org) as part of the greater [NameGuard](https://nameguard.io) solution to help protect the ENS community.
 * Passes **100%** of the [official validation tests](https://github.com/adraffy/ens-normalize.js/tree/main/validate) (validated automatically with pytest on Linux, MacOS, and Windows, see below for details).
 * Passes an [additional suite of further tests](/tools/updater/update-ens.js#L54) for compatibility with the official [Javascript reference implementation](https://github.com/adraffy/ens-normalize.js) and code testing coverage.
-* Based on [JavaScript implementation version 1.9.0](https://github.com/adraffy/ens-normalize.js/tree/4873fbe6393e970e186ab57860cc59cbbb1fa162).
+* Original implementation based on [JavaScript implementation version 1.9.0](https://github.com/adraffy/ens-normalize.js/tree/4873fbe6393e970e186ab57860cc59cbbb1fa162).
```

Comment:
```suggestion
* Supports updates to the ENS Normalize reference implementation for Unicode 16.0.
```

Spent some time thinking about this line. I think it's best we just fully remove the old idea / bullet point here. I agree it made since when we first released this library, but we already are making reference to Raffy's JS implementation up above and no benefit for anyone if we go into this detail here.

Alternatively, I think it's now useful to make a note here to identify which Unicode version we've updated this library to support.

Reply:
@lightwalker-eth I like the idea of mentioning unicode support. But what do you mean by "supports updates"?
Is it supposed to mean "Up-to-date with ENS Normalize reference implementation as of Unicode version 16.0"?
I'm happy to go with your version if you think it sounds ok.

Reply:
@Carbon225 Good catch. My suggestion could have been more clear. I like the way you proposed writing it:

"Up-to-date with ENS Normalize reference implementation as of Unicode version 16.0."